Author: A. Jesse Jiryu Davis <jesse@mongodb.com>
Description: Apply upstream patch to fix CDRIVER-2014
Index: mongo-c-driver/Makefile.am
===================================================================
--- mongo-c-driver.orig/Makefile.am
+++ mongo-c-driver/Makefile.am
@@ -55,9 +55,6 @@ EXTRA_DIST += \
 EXTRA_DIST += $(wildcard orchestration_configs/*/*)

 dist-hook:
-	-rm -rf $(distdir)/doc/html/.buildinfo $(distdir)/doc/html/.doctrees
-	-rm -rf $(distdir)/doc/man/.buildinfo $(distdir)/doc/man/.doctrees
-
 	@if test -d "$(srcdir)/.git"; then                                              \
 		(cd "$(srcdir)" &&                                                           \
       $(top_srcdir)/build/autotools/missing --run git log --stat ) > ChangeLog.tmp \
@@ -70,8 +67,6 @@ dist-hook:

 clean-local:
 	-rm -rf $(CLEANFILES)
-	-rm -rf doc/html/.buildinfo doc/html/.doctrees
-	-rm -rf doc/man/.buildinfo doc/man/.doctrees

 uninstall-local:
 	-rmdir $(mongocdocdir)/doc $(mongocdocdir)/html
Index: mongo-c-driver/doc/html/Makefile.am
===================================================================
--- mongo-c-driver.orig/doc/html/Makefile.am
+++ mongo-c-driver/doc/html/Makefile.am
@@ -19,5 +19,6 @@ doc/html: $(SPHINX_HTML_FILES)
 $(SPHINX_HTML_FILES):
 	-mkdir -p doc/html
 	$(SPHINX_BUILD) -qEW $(top_srcdir)/doc doc/html
+	rm -rf doc/html/.buildinfo doc/html/.doctrees

 EXTRA_DIST += doc/html
Index: mongo-c-driver/doc/man/Makefile.am
===================================================================
--- mongo-c-driver.orig/doc/man/Makefile.am
+++ mongo-c-driver/doc/man/Makefile.am
@@ -11,5 +11,6 @@ man3_MANS = $(wildcard doc/man/*.3)
 doc/man:
 	-mkdir -p doc/man
 	$(SPHINX_BUILD) -qEW -b man $(top_srcdir)/doc doc/man
+	rm -rf doc/man/.buildinfo doc/man/.doctrees

 EXTRA_DIST += doc/man
