Author: A. Jesse Jiryu Davis <jesse@mongodb.com>
Description: Whatis entry for mongoc_version man page
--- libmongoc-debian.orig/doc/mongoc_version.page
+++ libmongoc-debian/doc/mongoc_version.page
@@ -6,7 +6,8 @@
     <link type="guide" xref="index#api-reference" />
   </info>
 
-  <title>Version Checks</title>
+  <title>Version_Checks</title>
+  <subtitle>Conditional compilation based on mongoc version</subtitle>
 
   <p>
     The following preprocessor macros can be used to perform various checks based on the version of the library you are compiling against.
