--- mongo-c-driver.spec.orig	2019-08-27 21:43:20.721612865 +0000
+++ mongo-c-driver.spec	2019-08-27 22:28:57.276867708 +0000
@@ -181,7 +181,8 @@
 %{_includedir}/%{libname}-%{libver}
 %{_libdir}/%{libname}-%{libver}.so
 %{_libdir}/pkgconfig/%{libname}-*.pc
 %{_libdir}/cmake/%{libname}-%{libver}
+%{_libdir}/cmake/mongoc*
 %{_mandir}/man3/mongoc*
 
 %files -n libbson
@@ -195,7 +195,8 @@
 %doc src/libbson/NEWS
 %{_includedir}/libbson-%{libver}
 %{_libdir}/libbson*.so
 %{_libdir}/cmake/libbson-%{libver}
+%{_libdir}/cmake/bson*
 %{_libdir}/pkgconfig/libbson-*.pc
 %{_mandir}/man3/bson*
 
