--- mongo-c-driver.spec.orig	2020-02-20 15:21:09.502193247 -0500
+++ mongo-c-driver.spec	2020-02-20 15:28:44.349849073 -0500
@@ -38,7 +38,6 @@
 BuildRequires: mongodb-server
 BuildRequires: openssl
 %endif
-BuildRequires: cmake(mongocrypt)
 BuildRequires: perl-interpreter
 # From man pages
 BuildRequires: python3
@@ -124,7 +123,6 @@
 %endif
     -DENABLE_EXAMPLES:BOOL=OFF \
     -DENABLE_UNINSTALL:BOOL=OFF \
-    -DENABLE_CLIENT_SIDE_ENCRYPTION:BOOL=ON \
     .
 
 make %{?_smp_mflags}
@@ -191,7 +189,6 @@
 %{_includedir}/%{libname}-%{libver}
 %{_libdir}/%{libname}-%{libver}.so
 %{_libdir}/pkgconfig/%{libname}-*.pc
-%{_libdir}/cmake/%{libname}-%{libver}
 %{_libdir}/cmake/mongoc-%{libver}
 %{_mandir}/man3/mongoc*
 
@@ -205,7 +202,6 @@
 %doc src/libbson/NEWS
 %{_includedir}/libbson-%{libver}
 %{_libdir}/libbson*.so
-%{_libdir}/cmake/libbson-%{libver}
 %{_libdir}/cmake/bson-%{libver}
 %{_libdir}/pkgconfig/libbson-*.pc
 %{_mandir}/man3/bson*
